squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/html/scraper/HtmlScraper.java

```diff
@@ -349,12 +350,7 @@ private String replaceCommands(String s) {
               	arrayElements[0] = el;
               	elements = new Elements(arrayElements); 
               }else {
-            	  
-            	  if(cssQuery.startsWith("res(")) {
-            		  useResource = true;
-            		  cssQuery = cssQuery.substring(cssQuery.indexOf("(")+1, cssQuery.lastIndexOf(")") );
-            	  }
```

Comment:
Why did you remove this `if` code block?

Reply:
Added it back. It was missed during resolving merge conflicts.